#!/usr/bin/env python
PACKAGE = "train"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("front_axle_offset", int_t, 0, "Front axle offset", 207, 0, 1024);

gen.add("front_axle_scale", int_t, 0, "Front axle scale", 377, 0, 1024);

gen.add("rear_axle_offset", int_t, 0, "Rear axle offset", 322, 0, 1024);

gen.add("rear_axle_scale", int_t, 0, "Rear axle scale", 157, 0, 1024);


exit(gen.generate(PACKAGE, "train", "state_publisher"));
